styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -124,6 +125,7 @@ class KubernetesDockerRunner implements DockerRunner {
   static final String TRIGGER_TYPE = "STYX_TRIGGER_TYPE";
   static final String ENVIRONMENT = "STYX_ENVIRONMENT";
   static final String LOGGING = "STYX_LOGGING";
+  static final String STYX_EXECUTION_COUNTER = "STYX_EXECUTION_COUNTER";
```

Comment:
The variable could be called EXECUTION_TRIES to align with the other naming and also since its called tries in Styx instead of counter